frontend/src/pages/NewTeacher.vue

```diff
@@ -1,352 +1,350 @@
-                    </b-input>
+                <b-field label="English Proficiency" class="half-width">
+                  <b-select
+                    v-model="EnglishProficiency"
+                    placeholder="Select one"
+                    expanded
+                  >
+                    <option value="No">No (Unable to understand at all)</option>
+                    <option value="Little">
+                      Little (Able to understand simple words)
+                    </option>
+                    <option value="Simple">
+                      Simple (Able to speak full sentences)
+                    </option>
+                    <option value="Intermediate">
+                      Intermediate (Able to understand simple words)
+                    </option>
+                  </b-select>
                 </b-field>
-
-
-                <b-field label="Email Address" class="half-width">
-                    <b-input v-model="Email" type="string"
-                        value="">
-                    </b-input>
+              </b-field>
+
+              <b-field grouped>
+                <b-field label="Second Language" class="half-width">
+                  <b-autocomplete
+                    v-model="SecondLanguage"
+                    field="NativeLanguage"
+                    ref="secondlanguagevalue"
+                    :data="languages"
+                    placeholder="e.g. Tamil"
+                    @typing="filteredLanguageDataArray"
+                    @select="
+                      (option) =>
+                        (selected.SecondLanguage = option.NativeLanguage)
+                    "
+                  >
+                    <template slot="header">
+                      <a @click="showAddSecondLanguage">
+                        <span> Add new... </span>
+                      </a>
+                    </template>
+                  </b-autocomplete>
                 </b-field>
 
-
-                <b-field label="Source" class="half-width">
-                    <b-input
-                        v-model="source"
-                        placeholder="Optional">                 
-                    </b-input>
+                <b-field label="Language Proficiency" class="half-width">
+                  <b-select
+                    v-model="SecondLanguageProficiency"
+                    placeholder="Select one"
+                    expanded
+                  >
+                    <option value="No">No (Unable to understand at all)</option>
+                    <option value="Little">
+                      Little (Able to understand simple words)
+                    </option>
+                    <option value="Simple">
+                      Simple (Able to speak full sentences)
+                    </option>
+                    <option value="Intermediate">
+                      Intermediate (Able to understand simple words)
+                    </option>
+                  </b-select>
```

Comment:
TODO: update these options as specified in https://github.com/TechLadies/irrc-madwish/issues/85

Reply:
Noted, will update as part of a separate PR as discussed